kmpauth-google/src/commonMain/kotlin/com/mmk/kmpauth/google/GoogleUser.kt

```diff
@@ -5,7 +5,8 @@ package com.mmk.kmpauth.google
  */
 public data class GoogleUser(
     val idToken: String,
-    val accessToken:String?=null,
-    val displayName: String = "",
+    val accessToken:String? = null,
+    val email: String? = null,
+    val displayName: String? = null,
```

Comment:
don't want to break things with new update. So instead of nullable displayName, it should be non nullable as before,and should be set as empty string if it is null

Reply:
ok I see, I ll revert the update. 
I think that you should consider this update when making major version increase because hiding nullability is not a good practice, with the current state it's not possible to distinguish the case when id token doesn't contain name from the case when user has actually an empty name. It's not a big difference but in general it's easier to let clients to handle nullability :)

Reply:
yes, good point. For major I'll keep this in my mind. Thank you for pointing out that @doolle89